assets/controllers/login-controller.js

```diff
@@ -0,0 +1,44 @@
+// A Stimulus JavaScript controller file
+// https://stimulus.hotwired.dev
+// @see templates/security/login.html.twig
+// Leverage the Symfony Stimulus Bridge
+// https://github.com/symfony/stimulus-bridge
+// More info on Symfony UX
+//https://github.com/symfony/ux
+
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
```

Comment:
We could also make this controller lazy to show that off:

```suggestion
/*
 * The following line makes this controller "lazy": it won't be downloaded until needed
 * See https://github.com/symfony/stimulus-bridge#lazy-controllers
 */
/* stimulusFetch: 'lazy' */
export default class extends Controller {
```